float64 joint_00
float64 joint_01
float64 joint_02
float64 joint_03
float64 joint_04
float64 joint_05
geometry_msgs/Pose pick_pose
geometry_msgs/Pose place_pose
